### ISSUE TITLE
REQUEST: New membership for howardjohn

### ISSUE THREAD

**Issue by howardjohn** [istio/community#130]
### GitHub user id
- howardjohn

### Company affiliation
- Google

### Requirements
- [x] I have reviewed the [community membership guidelines](https://github.com/istio/community/blob/master/ROLES.md#member)
- [x] I have reviewed the [contribution guidelines](https://github.com/istio/community/blob/master/CONTRIBUTING.md)
- [x] I have enabled [2FA on my GitHub account](https://github.com/settings/security)
- [x] I have subscribed to [contributors](https://discuss.istio.io/c/contributors)
- [x] I have made multiple contributions to the project or community. Contributions may include, but are not limited to:
    - Authoring or reviewing PRs on GitHub
    - Filing or commenting on issues on GitHub
    - Contributing to working group or community discussions
- [x] I have two sponsors that meet the sponsorship requirements listed in the [community roles](https://github.com/istio/community/blob/master/ROLES.md#sponsor).
- [x] I have spoken to my sponsors ahead of this application, and they have agreed to +1 on this issue to confirm they agree to sponsor this application.

### List of sponsors
- @costinm
- @duderino 

### List of contributions to the Istio project
- PRs reviewed
- PRs merged
* https://github.com/istio/istio/pull/11795
* https://github.com/istio/istio/pull/11702
* https://github.com/istio/tools/pull/60
* https://github.com/istio/tools/pull/48
- Issues responded to
* https://github.com/istio/istio/issues/11700
* https://github.com/istio/istio/issues/11022
* https://github.com/istio/istio/issues/11762
* https://github.com/istio/istio.io/issues/3176
- Workgroup(s) I am involved with: Networking, Environments

### Slack access
- howardjohn@google.com

### **NOTE TO SPONSORS**

Please assess the list of contributions and provide a short few sentences as a comment to this issue why you approve
sponsorship, and include an `/lgtm` in the comment.

**Comment by mandarjog** [comment]
I support John's membership. He has several important PRs in tools for release qualification testing including one that streamlines prometheus metrics collection.
/lgtm

**Comment by duderino** [comment]
@howardjohn please describe the non-PR contributions made... stuff like running the long-running burning tests.   You can also refer to other PRs and bugs that reference the testing contributions.

Conditional approval by @louiscryan @geeknoid @linsun on additional pieces of documentation.    Need @rshriram and @dcberg @smawson approval offline

**Comment by howardjohn** [comment]
Sure. I have been working a lot on the long running performance tests (https://github.com/istio/tools). I have added multiple tests there so that we can measure the performance of new features like http 1.0, envoy proxy graceful shutdown, external traffic with the Sidecar resource. I have also worked on automation/monitoring of the tests - currently most of the process is pretty manual looking at metrics, but I have added tests that will pull from prometheus (https://github.com/istio/tools/pull/60) and am working on adding alarms.
